docs/quickstart.rst

```diff
@@ -23,28 +23,28 @@ To install the Python client library using pip:
 
 ::
 
-    pip install nexmo
+    pip install vonage
 
 Alternatively, you can clone the repository:
 
 ::
 
-    git clone git@github.com:Nexmo/nexmo-python.git
+    git clone git@github.com:Vonage/vonage-python.git
 
 Usage
 -----
 
-Begin by importing the nexmo module:
+Begin by importing the vonage module:
 
 .. code:: python
 
-    import nexmo
+    import vonage
 
 Then construct a client object with your key and secret:
 
 .. code:: python
 
-    client = nexmo.Client(key=api_key, secret=api_secret)
+    client = vonage.Client(key=api_key, secret=api_secret)
 
 For production, you can specify the ``NEXMO_API_KEY`` and
 ``NEXMO_API_SECRET`` environment variables instead of specifying the key
```

Comment:
Environment variables should change to VONAGE_API_KEY and VONAGE_API_SECRET respectively.